language/string_spec.rb

```diff
@@ -176,6 +176,13 @@
     s.should == '    foo bar#{@ip}' + "\n"
   end
 
+  ruby_version_is "2.3" do
+    it "allows HEREDOC with <<~'identifier', allowing to indent identifier and content" do
```

Comment:
I see, you meant the heredoc identifier, not the variable.
I think it's worth moving this to a fixture and under a method (say `StringSpec.squiggly`) to improve readability and make it a more real-world example.
You can then `require fixture(__FILE__, "squiggly_heredoc.rb")`